sanic_jwt/authentication.py

```diff
@@ -145,10 +144,13 @@ def _get_user_id(self, user):
 
     async def get_refresh_token(self, request, user):
         refresh_token = utils.generate_token()
-        user_id = self._get_user_id(user)
-        await self.store_refresh_token(user_id=user_id,
-                                       refresh_token=refresh_token,
-                                       request=request)
+        user_id = await self._get_user_id(user)
+        if inspect.iscoroutinefunction(self.store_refresh_token):
+            await self.store_refresh_token(
+                user_id=user_id, refresh_token=refresh_token, request=request)
+        else:
+            self.store_refresh_token(
```

Comment:
What about if `store_refresh_token` has not been implemented? I realize why you took the dummy out of the base class, but we should run a `hasattr` first, and if `None`:

    raise exceptions.RefreshTokenNotImplemented